qhub/provider/cicd/linter.py

```diff
@@ -0,0 +1,101 @@
+import os
+import json
+import textwrap
+import pathlib
+import subprocess
+import requests
+
+
+def qhub_validate():
+    # Gather the output of `qhub validate`.
+    print("Validate: info: validating QHub configuration in qhub-config.yaml")
+
+    command = "qhub validate qhub-config.yaml"
+    validate_output = subprocess.run([command], shell=True, capture_output=True)
```

Comment:
I'd prefer to not use subprocess. See this for catching errors https://github.com/Quansight/conda-store/blob/5bf691068561986756740a0baab1ee035ffbeffb/conda-store-server/conda_store_server/build.py#L216-L220.